packages/core/src/Stores/client-store.js

```diff
@@ -1782,7 +1781,9 @@ export default class ClientStore extends BaseStore {
                 utm_campaign: ppc_campaign_cookies?.utm_campaign,
                 utm_content: ppc_campaign_cookies?.utm_content,
                 domain: window.location.hostname,
-            });
+            };
+            if (this.user_id) analytics_config.user_id = this.user_id;
```

Comment:
To make sure that we are not passing empty string to analytics package as it can override the current one.

Reply:
`  country: Cookies.get('clients_country') || Cookies?.getJSON('website_status')?.clients_country,`
This is called in the analytics right.we can remove it from here no?

Reply:
For that I need to bump up the analytics package again.
Because its in the newer version of the package.